### PR TITLE
Add REFRESH_REQUESTED connection state

### PR DESCRIPTION
**Pull request by glazychev-art**
Issue: https://github.com/networkservicemesh/cmd-nse-vl3-vpp/issues/77
Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

### CODE REVIEW

File: pkg/api/ipam/ipam.pb.go [hunk 1]
```diff
@@ -18,7 +18,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.17.3
```
Comment (denis-tingaikin) [inline pkg/api/ipam/ipam.pb.go:21]: Is this change expected? Can we be consistent with other pieces?

File: pkg/api/networkservice/connection.pb.go [hunk 2]
```diff
@@ -46,19 +46,22 @@ const (
...
 		0: "UP",
 		1: "DOWN",
+		2: "REFRESH_REQUESTED",
 	}
 	State_value = map[string]int32{
-		"UP":   0,
-		"DOWN": 1,
+		"UP":                0,
+		"DOWN":              1,
+		"REFRESH_REQUESTED": 2,
```
Comment (denis-tingaikin) [inline pkg/api/networkservice/connection.pb.go:64]: Hmm.. What about that?

```suggestion
		"REFRESHING": 2,
```

Comment (glazychev-art) [reply pkg/api/networkservice/connection.pb.go:64]: @edwarnicke 
What do you think?

Comment (edwarnicke) [reply pkg/api/networkservice/connection.pb.go:64]: I think it really is 'REFRESH_REQUESTED' ... 'REFRESHING' implies that the refresh is actively happening now... which it isn't yet, until the NSC sends a refresh 'Request'.

**Review by denis-tingaikin — APPROVED** [review]
@edwarnicke Feel free to merge this

**Pull request merged**